README.md

```diff
@@ -142,6 +142,58 @@ to `latest`:
 ORT_SERVER_IMAGE_PREFIX= ORT_SERVER_IMAGE_TAG=latest docker compose up
 ```
 
+#### Handling major version upgrades of PostgreSQL
+
+PostgreSQL does not support reading the data directory of previous major versions.
+When upgrading the PostgreSQL version in the Docker Compose setup, the database data directory must be manually
+migrated by following these steps:
```

Comment:
Shouldn't this be a single line according to conventions?

Reply:
This file was not yet migrated to use one sentence per line so I wanted to keep it consistent.